### PR TITLE
add client class to projects model and add fixture file with all poss…

### PR DESCRIPTION
**Pull request by batemapf**
add client class to projects model and add fixture file with all possible clients. also added foreignkey to project class. also added client class to admin.

TO DO: ~need to update readme with loaddata instruction~

### CODE REVIEW

**Comment by batemapf** [comment]
added commit for readme update

File: projects/admin.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from django.contrib import admin
 
-from .models import Project
+from .models import Project, Client
```
Comment (brendansudol) [inline projects/admin.py]: nit and personal preference: how do you feel about alphabetizing these imports (as well as on lines 5-6)? I tend to like the consistency of that pattern, but feel free to ignore :)

File: projects/models.py [hunk 2]
```diff
@@ -8,7 +8,22 @@ class ModelBase(models.Model):
...
+    omb_agency_code = models.CharField(max_length=255, blank=True)
+    omb_bureau_code = models.CharField(max_length=255, blank=True)
+    treasury_agency_code = models.CharField(max_length=255, blank=True)
+    cgac_agency_code = models.CharField(max_length=255, blank=True)
+
+    def __str__(self):
+        return '%s - %s' % (self.department, self.agency)
+
+    class Meta:
+        verbose_name = 'Client'
```
Comment (brendansudol) [inline projects/models.py]: what's the thinking behind the capitalization here? i've never done that before, but happy to override default if it's advantageous

**Comment by brendansudol** [comment]
thanks for adding this @batemapf -- this is awesome; one small thing: we're using flake8 as our python style linter (I still need to wire this up to codeclimate) and it's flagging two tiny things:

```
(venv)projects $ flake8 
./projects/models.py:11:1: E302 expected 2 blank lines, found 1
./projects/models.py:26:1: E302 expected 2 blank lines, found 1
```

(you can run on entire repo with `flake8` command and can do individual files too `flake8 FILE_PATH`). you mind making these tweaks?

File: projects/models.py (hunk 2 above)
Comment (batemapf) [inline projects/models.py]: not following. is something capitalized or not capitalized that you expected to be different?

Comment (brendansudol) [inline projects/models.py]: sorry, lines 22-24 -- you're overriding the default verbose_name (https://docs.djangoproject.com/en/1.9/ref/models/options/#verbose-name), not sure we need to

Comment (batemapf) [inline projects/models.py]: ah! wanted to make sure that the `Client` class was displayed correctly in the admin panel, specifically on the front page of the panel it was specified as `Clients`. i dropped out the `Meta` sub-class and it still looks fine, so looks like it was unnecessary :-).

**Comment by batemapf** [comment]
addressed each of the line notes and ran the code through flake8 + made corrections. _should_ be good.

**Comment by brendansudol** [comment]
LGTM!

**Pull request merged**